DESCRIPTION

```diff
@@ -1,17 +1,18 @@
 Package: admiralpeds
 Type: Package
-Title: ADaM in R Asset Library - Extension
-Version: 0.0.1
+Title: Pediatrics Extension Package for ADaM in 'R' Asset Library
+Version: 0.1.0
 Authors@R: c(
-    person("Open", "Source", email = "admiraltemplate@pharmaverse.com", role = c("aut", "cre")))
+    person("Ross", "Farrugia", email = "ross.farrugia@roche.com", role = c("aut", "cre")))
 Description: A toolbox for programming Clinical Data Standards Interchange Consortium
     (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a
     mandatory part of any New Drug or Biologics License Application submitted to the
     United States Food and Drug Administration (FDA). Analysis derivations are
     implemented in accordance with the "Analysis Data Model Implementation Guide"
-    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam/adamig-v1-3-release-package>).
+    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam>).
```

Comment:
Do we still need the year here?  Wondering if we should drop that here and in other admiral packages?

Reply:
yes could probably be dropped - maybe best to have one person go through all and drop this in one go